### PR TITLE
changes to support for ruby 1.9.3

### PR DESCRIPTION
**Pull request by geekguy**

### CODE REVIEW

File: shoryuken.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rspec'
-  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'pry-debugger'
```
Comment (phstc) [inline shoryuken.gemspec]: @geekguy hm this one breaks the compatibility with Ruby 2.0. If we keep `pry-byebug` as `add_development_dependency` can you still run Shoryuken with 1.9.3?

File: lib/shoryuken/fetcher.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def receive_messages(queue, limit)
       # AWS limits the batch size by 10
       limit = limit > FETCH_LIMIT ? FETCH_LIMIT : limit
 
-      options = Shoryuken.options[:aws][:receive_message].to_h.dup
+      options = (Shoryuken.options[:aws][:receive_message] || {}).to_hash.dup
```
Comment (phstc) [inline lib/shoryuken/fetcher.rb]: Not sure if `to_hash` is needed here, as it returns `self`.

Comment (geekguy) [inline lib/shoryuken/fetcher.rb]: @phstc : Are we using ```to_h```  only to convert ```nil``` to empty hash?

Comment (phstc) [inline lib/shoryuken/fetcher.rb]: @geekguy correct, just that.

Comment (geekguy) [inline lib/shoryuken/fetcher.rb]: @phstc : I have removed `to_hash`.

**Comment by phstc** [comment]
Alright @geekguy. Added to master :beers: 

The only change I  reverted was the `pry-debugger` one, as it breaks compatibility with Ruby 2.0.0+.

**Pull request merged**